src/pickup.cpp

```diff
@@ -83,6 +83,9 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
     const bool can_be_folded = veh->is_foldable();
     const bool is_convertible = ( veh->tags.count( "convertible" ) > 0 );
     const bool remotely_controlled = g->remoteveh() == veh;
+    const bool has_washmachine = ( veh->part_with_feature( veh_root_part, "WASHING_MACHINE" ) >= 0 );
+    bool working = false;
+    bool detergent_is_enough = false;
```

Comment:
Move this variable into the case.
You can have variable declarations inside a case if you wrap the block in braces, like this:
```
switch( x ) {
    case y: {
        bool is_happening = true;
        if( is_happening ) add_msg( m_good, "It's happening!" );
    }
}
```